modules/nf-core/lofreq/somatic/tests/main.nf.test.snap

```diff
@@ -28,21 +76,20 @@
         ],
         "meta": {
             "nf-test": "0.8.4",
-            "nextflow": "24.04.2"
+            "nextflow": "23.04.0"
         },
-        "timestamp": "2024-07-10T10:37:58.931241788"
+        "timestamp": "2024-07-18T08:08:38.275671964"
     },
-    "human - bam": {
+    "version": {
         "content": [
-            10,
```

Comment:
Previously there were 10 files in this channel, what has happened to the others?

Reply:
Lofreq somatic produces several files, most of them can be ignored according to [docs](https://csb5.github.io/lofreq/commands/#somatic) since, they are not the "final" files. These "final" files are the ones called `somatic_final.snvs.vcf.gz ` and `somatic_final.indels.vcf.gz` and these are the files that are included as outputs now (with their respective `.tbi` files)